kevm

```diff
@@ -54,11 +54,18 @@ run_krun() {
     local run_file=$1 ; shift
     run_env "$run_file"
     export K_OPTS=-Xss500m
-    if [[ "$backend" == "haskell" ]]; then
-        krun --directory "$build_dir/$backend/" -cSCHEDULE="$cSCHEDULE" -pSCHEDULE='printf %s' -cMODE="$cMODE" -pMODE='printf %s' "$run_file" "$@" --haskell-backend-command "$build_dir/kore/bin/kore-exec"
-    else
-        krun --directory "$build_dir/$backend/" -cSCHEDULE="$cSCHEDULE" -pSCHEDULE='printf %s' -cMODE="$cMODE" -pMODE='printf %s' "$run_file" "$@"
-    fi
+    case "$backend" in
+        haskell)
+            args=(--haskell-backend-command $build_dir/kore/bin/kore-exec)
+            ;;
+        ocaml)
+            args=(--interpret)
+            ;;
+        java)
+            args=(--z3-executable)
```

Comment:
This has been deprecated on the newest `master` (--z3-executable is no longer an option, it's default).